typescript/cli/src/deploy/core.ts

```diff
@@ -201,7 +245,8 @@ interface DeployParams {
   signer: ethers.Signer;
   multiProvider: MultiProvider;
   artifacts?: HyperlaneAddressesMap<any>;
-  multisigConfig?: ChainMap<MultisigConfig>;
+  ismConfigs?: ChainMap<IsmConfig>;
+  multisigConfigs?: ChainMap<MultisigConfig>;
```

Comment:
can we generate a `ChainMap<IsmConfig>` from the multisig config?

Reply:
for the basic variant, yes